### PR TITLE
caching the cargo related files can help build speeds a lot

### PR DESCRIPTION
**Pull request by Connoropolous**
don't need the nodejs thing because nix is coming with node

i was working on improving all this over in 
https://github.com/lightningrodlabs/electron-holochain-template

could also add caching of node_modules with pnpm-lock file as the cache busting key

### CODE REVIEW

**Comment by shanedeeodonnell** [comment]
@Connoropolous sweet thanks! Might as well cache the node_modues while we're at it. Makes sense to me.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -25,10 +25,19 @@ jobs:
         uses: cachix/install-nix-action@v16
         with:
           nix_path: nixpkgs=channel:nixos-21.11
-      - name: Use Node.js ${{ matrix.node-version }}
```
Comment (adaburrows) [inline .github/workflows/release.yml:28]: Oh, we never needed this, did we? We get it with our nix config.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -25,10 +25,19 @@ jobs:
...
-        uses: actions/setup-node@v1
+      - uses: actions/cache@v2
```
Comment (adaburrows) [inline .github/workflows/release.yml:28]: Nice! Should we also cache node_modules?

File: .github/workflows/release.yml (hunk 1 above)
Comment (Connoropolous) [reply .github/workflows/release.yml:28]: Thats right. I was reviewing our pattern and realized it's unnecessary

File: .github/workflows/release.yml (hunk 2 above)
Comment (Connoropolous) [reply .github/workflows/release.yml:28]: Sounds good. Go with this PR as is and mark node npm followups

**Pull request merged**